src/app/core/components/group-nav/group-nav.component.ts

```diff
@@ -20,10 +21,12 @@ export class GroupNavComponent {
 
   joinedGroups: GroupData = [];
   managedGroups: GroupData = [];
+  selectedGroup: number;
```

Comment:
"0" for yourself while I expect a selected group (so a group id) is misleading. If you just need it for yourself for now, use a boolean.

Reply:
This variable is used to highlight which group tab has been selected, the variable name makes in the file (the tabs are called group-self, group-joined, and group-managed) but I can see why it can be misleading.

How about renaming it to selectedTab ? Also, this variable isn't currently used in the other element, but according to the design it should be, should I add it in another PR or in this one ?